### PR TITLE
Fix IDE0019 pattern matching issue

### PR DESCRIPTION
**Pull request by chlafreniere**
<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>

wasn't included in all of the .csproj files.

Happy to split that out if we want into a separate PR. Could also put this in a .props file, but that seemed a bit silly for one property. Happy to hear opinions here.

Finally, I'm unclear why it shows every line as changed. Is that because line endings are defined as clrf? It only shows a couple of lines as changed for me in VS Code:

image

### CODE REVIEW

File: test/SqlExtension.Tests/TestData.cs [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-                string.Equals(this.Name, otherData.Name, StringComparison.OrdinalIgnoreCase)) && this.Timestamp.Equals(otherData.Timestamp);
-        }
-    }
-}
+﻿using System;
+
+namespace SqlExtension.Tests
+{
+    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
+    public class TestData
```
Comment (chlafreniere) [inline test/SqlExtension.Tests/TestData.cs:6]: Could all of these changes be related to line endings?

File: test/SqlExtension.Tests/TestData.cs [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-            return this.ID == otherData.ID && this.Cost == otherData.Cost && ((this.Name == null && otherData.Name == null) ||
-                string.Equals(this.Name, otherData.Name, StringComparison.OrdinalIgnoreCase)) && this.Timestamp.Equals(otherData.Timestamp);
-        }
-    }
-}
+﻿using System;
+
+namespace SqlExtension.Tests
+{
+    #pragma warning disable CS0659 // Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
```
Comment (Charles-Gagnon) [inline test/SqlExtension.Tests/TestData.cs:5]: We should be implementing GetHashCode as well https://stackoverflow.com/questions/371328/why-is-it-important-to-override-gethashcode-when-equals-method-is-overridden

File: test/SqlExtension.Tests/TestData.cs (hunk 1 above)
Comment (Charles-Gagnon) [reply test/SqlExtension.Tests/TestData.cs:5]: Seems likely...I assume you see them as `\n` locally?

Comment (chlafreniere) [reply test/SqlExtension.Tests/TestData.cs:5]: https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/testeroni

Definitely a line ending thing. Personally, I don't care either way. Any strong opinions on how we should enforce this (if at all?)?

Comment (Charles-Gagnon) [reply test/SqlExtension.Tests/TestData.cs:5]: I think we can remove that - github takes care of that stuff for us generally (and if there are files that are for specific OSs such as .bat files then we can control that separately through gitattributes like vs code does https://github.com/microsoft/vscode/blob/main/.gitattributes)

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**